.github/workflows/run_acceptance_tests.yml

```diff
@@ -20,21 +20,31 @@ jobs:
         uses: ./client-engineering-deployment-tools/actions/mock-server
```

Comment:
Smaller PRs with tight scope next time please ;)

Reply:
Yes. Sorry for that. I did trim a lot if you compare it with #221 :)